package.json

```diff
@@ -165,6 +165,18 @@
                 "category": "Azure Resource Groups",
                 "icon": "$(refresh)"
             },
+            {
+                "command": "azureResourceGroups.focusGroup",
+                "title": "Focus Group",
```

Comment:
Would like PM input on this too, but maybe "Focus this Group"? To me, "Focus Group" sounds like a noun before I realize it's meant to be an verb/noun combo.